packages/std/src/query/mod.rs

```diff
@@ -35,6 +38,8 @@ pub enum QueryRequest<C> {
     Custom(C),
     #[cfg(feature = "staking")]
     Staking(StakingQuery),
+    #[cfg(all(feature = "staking", feature = "cosmwasm_1_3"))]
```

Comment:
Let's remove the staking feature gate here. The Distribution module also manages the community pool by moving issues coins from x/mint to stakers and tax to pool. It is not entirely clear to me how the two are coupled or independent.

I think the query does no harm even on chains without staking/distribution available. There it just fails.